### PR TITLE
Add Target (#266)

### PR DESCRIPTION
**Pull request by catd825**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Added [target.md](https://github.com/catd825/companies-on-blm/blob/target/companies/target.md) and [target.png](https://github.com/catd825/companies-on-blm/blob/target/images/target.png)

Fixes # [266](https://github.com/DiversityCorp/companies-on-blm/issues/266)

## Type of change

Please delete options that are not relevant.

- [x] Company data addition
- [ ] Bug fix
- [x] New feature

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:
* Browser:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: companies/target.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    raw_text: >
+      In a recent note, CEO Brian Cornell shared the Target team’s commitment to stand against racism. We pledged to face anxiety, fear and sorrow with purpose, and formed an enterprise task force to determine how to help create solutions for the injustice Black families and people of color face every day. 
+
+      Today, our next steps include announcing a $10 million commitment and ongoing resources to advance social justice and support rebuilding and recovery efforts in local communities. 
+
+      “Target stands with Black families, communities and team members. As we face an inflection point in Minneapolis and across the country, we’re listening to our team, guests and communities, committed to using our size, scale and resources to help heal and create lasting change,” says Brian Cornell, chairman and CEO, Target.
+
+      Initial efforts include:
+
+      A $10 million investment from Target and the Target Foundation to support long-standing partners such as the National Urban League and the African American Leadership Forum. We will also add new partners in the Minneapolis-St. Paul area and across the country.
```
Comment (kelseyhuse30) [inline companies/target.md:23]:
```suggestion
      A $10 million investment from Target and the Target Foundation to support long-standing partners such as the National Urban League and the African American Leadership Forum. We will also add new partners in the Minneapolis-St. Paul area and across the country.
```

A few "trailing spaces" errors.  Just means erase the space(s) at the end of the line to clean it up :)

**Review by kelseyhuse30 — APPROVED** [review]
Excellent @catd825 ! You found it all!! There are a few automated comments from our linter bot.  If you don't mind cleaning up those 3 (just remove the extra spaces from the end of the lines), then I will merge it in :)

Again, welcome to the project!

**Comment by catd825** [comment]
Awesome, think I got them all this time :) lmk if anything else is needed!

**Review by kelseyhuse30 — APPROVED** [review]
Thanks for cleaning up the lint errors! Looks beautiful now :D

**Pull request merged**